web/cobrands/sass/_base.scss

```diff
@@ -2499,8 +2499,16 @@ label .muted {
 }
 
 a#geolocate_link {
-  display: inline-block;
-  padding: 0.5em;
+  display: inline-block !important;
+  border-radius: 6px;
+  border: 1px solid #333333;
+  color: #222222;
+  text-decoration: none;
+  font-weight: bold;
+  background: transparent url("/i/locate-me-arrow-black.svg") 4% 50% no-repeat;
+  padding: 15px 15px 15px 40px;
```

Comment:
The core styles in fixmystreet are designed to work in both left-to-right (LTR) and right-to-left (RTL) languages, and we tend to make sure things like horizontal paddings and margins switch to suit the language direction.

We use a Sass function called `flip()` for this. Do a quick search of the Sass code for `flip(` to see examples of how it works. Give us a shout if it’s not obvious.